ui/admin/app/components/form/role/grants/index.hbs

```diff
@@ -13,20 +13,20 @@
     <Rose::List::KeyValue as |list|>
       {{! Add grant field (only one per form) }}
       <list.item as |item|>
-        <form.input
-          @contextual={{true}}
-          @name='grant'
-          @type='text'
-          @value={{this.newGrantString}}
-          as |input|
-        >
+        <form.input @contextual={{true}}>
           <item.key>
-            <input.label>{{t
-                'resources.role.grant.actions.create'
-              }}</input.label>
+            <Hds::Form::Label @controlId='add-grant'>
```

Comment:
Note: In order to honor the form layout of `Rose::List::KeyValue` I kept the `Hds::Form::Label` component inside of `<item.key>` and used `@controllerId` to connect the label to the `Input` component. This changed the form field specs, so I added `class='rose-form-input-field'` to `Input` to ensure the text fields have identical specs